src/hotspot/share/oops/constantPool.hpp

```diff
@@ -77,6 +77,36 @@ class CPKlassSlot {
   }
 };
 
+class BSMAttributeEntry {
```

Comment:
Please consider moving the description comment from lines #569-#572 ahead of this structure

Reply:
This?

```
  // The first part of the operands array consists of an index into the second part.
  // Extract a 32-bit index value from the first part.
```

Wouldn't it be better to leave it where it is, but add a comment ahead of the structure?